glossary.rst

```diff
@@ -121,7 +121,7 @@ Glossary
 
    YAML
         *YAML* is a recursive acronym for "YAML Ain't a Markup Language". It's a
-        lightweight, humane data serialization language used extensively in
+        lightweight, human data serialization language used extensively in
```

Comment:
Actually, I think the initial version is right (humane means something like "human friendly" or "easy to understand/read for humans").